docs/wget2_manual.md

```diff
@@ -673,6 +673,19 @@ Go to background immediately after startup. If no output file is specified via t
   Force Wget2 to unlink file instead of clobbering existing file. This option is useful for downloading to the
   directory with hardlinks.
 
+* --cut-url-get-vars
+
+  Remove HTTP GET Variables from URLs.
+  For example "main.css?v=123" will be changed to "main.css".
+  Be aware that this may have unintended side effects, for example "image.php?name=sun" will be changed 
+  to "image.php". The cutting happens before adding the URL to the download queue.
+
+* --cut-file-get-vars
+
+  Remove HTTP GET Variables from filenames.
+  For example "main.css?v=123" will be changed to "main.css".
+  Be aware that this may have unintended side effects, for example "image.php?name=sun" will be changed 
+  to "image.php". The cutting happens when saving the file, after downloading.
```

Comment:
A small segment here about how it works along with `--trust-server-names` and `--content-disposition` would be nice. Especially a hint that using `--content-disposition` _may_ prevent the side effects that are shown above